### ISSUE TITLE
Add term displaced anterior fontanelle

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10554]
**Preferred term label:**
Displaced anterior fontanelle

**Synonyms**
-
**Definition (free text, please give PubMed ID)**
Displacement of the anterior fontanelle due to abnormality of the cranial sutures, mostly craniosynostosis. PMID: 31691965, PMID: 20216454

**Parent term (use hpo.jax.org/app)**
Abnormality of the anterior fontanelle HP:0000236

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Craniofacial syndromes. For example Cranioectodermal dysplasia (omim613610) PMID24123776

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6001069
- label: Displaced anterior fontanelle
- parents: [Abnormality of the anterior fontanelle (HP:0000236)](https://hpo.jax.org/app/browse/term/HP:0000236)
- pmid: PMID:20216454|PMID:31691965|PMID:24123776
- def: Abnormal shifting of the anterior fontanelle fropm its normal central position.
- comment: This feature can be related to an abnormality of the cranial sutures, mostly craniosynostosis.
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10554
- synonyms: Displacement of the anterior fontanelle(NONE)
